README.md

```diff
@@ -5,3 +5,15 @@
 [![Discussion Forum](https://img.shields.io/discourse/https/forum.vaticle.com/topics.svg)](https://forum.vaticle.com)
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-typedb-796de3.svg)](https://stackoverflow.com/questions/tagged/typedb)
 [![Stack Overflow](https://img.shields.io/badge/stackoverflow-typeql-3dce8c.svg)](https://stackoverflow.com/questions/tagged/typeql)
+
+This repository stores all TypeDB Drivers built by Vaticle.
+
+See the table below for links to README files, documentation, and source codes.
+
+| Driver  | Readme                                                                             | Documentation                                                                 | Source codes                                                                                         |
+|---------|------------------------------------------------------------------------------------|-------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|
+| Rust    | [README](https://github.com/vaticle/typedb-driver/tree/development/rust/README.md) |                                                                 | `rust/src` [directory](https://github.com/vaticle/typedb-driver/tree/development/rust/src)           |
+| Python  | [README](https://github.com/vaticle/typedb-driver/tree/development/python/README.md) | [Documentation](https://typedb.com/docs/clients/2.x/python/python-overview)   | `python/typedb` [directory](https://github.com/vaticle/typedb-driver/tree/development/python/typedb) |
+| Java    | [README](https://github.com/vaticle/typedb-driver/tree/development/java/README.md) | [Documentation](https://typedb.com/docs/clients/2.x/java/java-overview)       | `java` [directory](https://github.com/vaticle/typedb-driver/tree/development/java)                   |
+| Node.js | [README](https://github.com/vaticle/typedb-driver/tree/development/nodejs/README.md) | [Documentation](https://typedb.com/docs/clients/2.x/node-js/node-js-overview) | `nodejs` [directory](https://github.com/vaticle/typedb-driver/tree/development/nodejs)               |
+| C       |    |                                                                               | `c/src` [directory](https://github.com/vaticle/typedb-driver/tree/development/c/src)                 |
```

Comment:
Please verify that this is the intended ordering of the five languages.

Reply:
Double-checked with Haikal, and he adjusted the order. Fixed in https://github.com/vaticle/typedb-driver/pull/453/commits/395640f51a726a661f11f2a2531c8fbb62bbaaea